lib/flb_libco/aarch64.c

```diff
@@ -12,8 +12,10 @@
 #include <string.h>
 #include <stdint.h>
 
+#ifndef __APPLE__
 #ifndef IOS
-#include <malloc.h>
+#include <malloc/malloc.h>
```

Comment:
ugh this is a typo.
it shouldn't have changed to malloc/malloc.h:
https://github.com/monkey/monkey/pull/340/files